packages/react-components/src/templates/GroupProfileAbout.tsx

```diff
@@ -59,17 +57,18 @@ const GroupProfileAbout: React.FC<GroupProfileAboutProps> = ({
         active={active}
       />
       <div id={membersSectionId}>
-        {active ? (
-          <GroupMembersSection teams={teams} leaders={leaders} />
-        ) : (
-          <div css={membersSectionStyles}>
-            <LeadersTabbedCard
-              title="Interest Group Leaders"
-              leaders={leaders}
-            />
-            <TeamsTabbedCard title="Interest Group Teams" teams={teams} />
-          </div>
-        )}
+        <div css={membersSectionStyles}>
+          <GroupLeadersTabbedCard
+            title="Interest Group Leaders"
+            leaders={leaders}
+            isGroupInactive={!active}
+          />
+          <GroupTeamsTabbedCard
+            title="Interest Group Teams"
+            teams={teams}
+            isGroupInactive={!active}
```

Comment:
```suggestion
            isGroupActive={active}
```
Suretly this is better?